games/r.yaml

```diff
@@ -122,16 +122,17 @@
   type: remake
   originals:
   - Redneck Rampage
+  - Duke Nukem 3D
   repo: https://github.com/nukeykt/NBlood
-  url: https://forums.duke4.net/topic/10241-rednukem-redneck-rampage-port/
+  url: https://nukeykt.retrohost.net/
   development: sporadic
```

Comment:
Still sporatic? It is very active.

Reply:
I changed it, it didn't get picked up.  1 sec.  I'm new to contributing so bear with me please.

Reply:
No problem, thanks for everything. I wanted to open issues for these ports 1 minute before I saw your PRs. Did you see this repo on Hacker News too?

Reply:
Nope, must just be a total coincidence.   I was messing around with PCExhumed on my Pinebook Pro (ARM laptop).. then saw a link to your website on Reddit.  Then I saw that Rednukem also wasn't in the list, so I figured I would add both